src/shared/organisms/DataPanel/DataPanel.tsx

```diff
@@ -467,7 +486,7 @@ const DataPanel: React.FC<Props> = ({}) => {
                 : 'Resource',
             resourceId: resource.id,
             project: `${parsedSelf.org}/${parsedSelf.project}`,
-            path: `/${parsedSelf.project}/${pathId}${
+            path: `/${parsedSelf.project}/${parsedSelf.id}/${pathId}${
```

Comment:
**A note** on why I changed the `path`.

In my-data page in [staging](https://staging.nise.bbp.epfl.ch/nexus/) the download by selecting all rows in the table is not working because (according to the server response) the path is repeated for some resources that are being downloaded.

If we do a text search (ctrl + F) on the page, we see that there are 2 rows with column "name" = organization (this is one example). Apparently their project names are also same so the path was getting repeated. Adding the `id` obtained from `self` to the path makes the paths unique and fixes the problem.

I don't know if this should happen in production too. Let me know if you foresee issues with this solution.

Reply:
Yeah, that was a good catch. I've fixed the path length issue in PR for [3898](https://github.com/BlueBrain/nexus-web/pull/1250).